### DISCUSSION TITLE
Copy and paste SVG's from clipboard

### DISCUSSION THREAD

**Discussion by girafic** [uxbox/uxbox#516] [Ideas]
I think it's already on the roadmap. So, copy and paste SVG images from clipboard would be nice. At the moment it's as text there.

![Bildschirmfoto 2021-02-02 um 12 13 39](https://user-images.githubusercontent.com/1187276/106592793-396a7800-6550-11eb-8793-15ff396b13c2.jpg)

**Comment by Alotor** [comment]
I know it's not ideal but if you put it on a file with .svg extension and then just drag/drop from your explorer into the application it will work.

We want to make a lot of improvements on the SVG importer, this will be added for sure. 

Thanks for your feedback :smiley:

**Comment by girafic** [comment]
Discussion can be closed or deleted. Feature is already implemented.
